site/content/features/gatherers.html

```diff
@@ -121,23 +121,23 @@ <h2>A Reminder About Collectors</h2>
 <div class='sandbox'>    <pre>void main() {
     Map&lt;String, Map&lt;String, List&lt;Locale&gt;&gt;&gt; countryAndLanguageToLocale =
         Locale.availableLocales().collect(
-            groupingBy(Locale::getCountry,
-                groupingBy(Locale::getLanguage)));
+            Collectors.groupingBy(Locale::getCountry,
+                Collectors.groupingBy(Locale::getLanguage)));
     println(countryAndLanguageToLocale.get("IN").get("hi"));
 }
 </pre></div>
       <p>Now you have a map of maps. For example, <code>countryAndLanguageToLocale.get("IN").get("hi")</code> is a list of the Hindi locales in India. (There are several variants.)</p>
       <h2>The Gatherer Interface</h2>
       <p>JEP 485 introduces a <code>gather</code> method that is analogous to the <code>collect</code> method, but it is an intermediate operation, yielding a stream, not a final result. The method has an argument of type <code>Gatherer</code>, an interface that is similar to the <code>Collector</code> interface. A gatherer has a generic “intermediate state” object instead of the “result collection” of a collector.  It too has four methods yielding function objects. </p>
-      <pre>Supplier&lt;A&gt; initializer()
-    // Supplies an intermediate state instance
-Gatherer.Integrator&lt;A, T, R&gt; integrator()
-    // Integrates an element—see below for details
-BinaryOperator&lt;A&gt; combiner()
-    // Combines two intermediate states into one
-BiConsumer&lt;A, Gatherer.Downstream&lt;? super R&gt;&gt; finisher()
-    // Finalizes the processing after all stream elements have been integrated
-</pre>
+      <dl><dt><code>Supplier&lt;A&gt; initializer()</code></dt>
+        <dd>Supplies an intermediate state instance</dd>
+        <dt><code>Gatherer.Integrator&lt;A, T, R&gt; integrator()</code></dt>
+        <dd>Integrates an element—see below for details</dd>
+        <dt><code>BinaryOperator&lt;A&gt; combiner()</code></dt>
+        <dd>Combines two intermediate states into one</dd>
+        <dt><code>BiConsumer&lt;A, Gatherer.Downstream&lt;? super R&gt;&gt; finisher()</code></dt>
+        <dd>Finalizes the processing after all stream elements have been integrated</dd>
+      </dl>
       <p>This is more complex than collectors since the gatherer sends values “downstream”; that is, to the stream that is the result of the <code>gather</code> method. </p>
       <p>Let's first look at the <code>finish</code> method. It is called after all upstream elements have been passed to the <code>integrator</code> function, and all <code>combiner</code> functions have been invoked. We now have the final version of the “intermediate state”. From it, the <code>finisher</code> method can derive zero or more values and send them downstream, using the methods of the <code>Gatherer.Downstream</code> interface:</p>
       <pre>boolean push(T element)
```

Comment:
Sorry, here is another one that may better use the `dl` formatting.

Reply:
Sorry, which is that?

Reply:
The ones after this phrase: "the methods of the Gatherer.Downstream interface:"

Reply:
It already uses dl.

Reply:
Sorry, my bad. Didn't update my local copy correctly as I was working on the script.